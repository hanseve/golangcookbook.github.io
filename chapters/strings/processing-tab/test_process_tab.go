package main

import (
	"fmt"
	"regexp"
	"strings"
)

func main() {
	fmt.Printf("%q\n",strings.ReplaceAll("oink\toink\toink", "\t", ""))
	fmt.Printf("%q\n",strings.ReplaceAll("foo foo foo", " ", "\t"))

	re := regexp.MustCompile("\t")
	fmt.Printf("%q\n",re.ReplaceAllString("oink\toink\toink", ""))
	re = regexp.MustCompile(" ")
	fmt.Printf("%q\n",re.ReplaceAllString("foo foo foo", "\t"))
}
