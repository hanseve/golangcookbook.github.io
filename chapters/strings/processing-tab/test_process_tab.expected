"oinkoinkoink"
"foo\tfoo\tfoo"
"oinkoinkoink"
"foo\tfoo\tfoo"
